crmprtd/ec.py

```diff
@@ -29,20 +30,25 @@ def makeurl(freq='daily', province='BC', language='e', time=datetime.utcnow()):
     """
     fmt = '%Y%m%d%H' if freq == 'hourly' else '%Y%m%d'
     freq = 'yesterday' if freq == 'daily' else freq
-    fname = '%s_%s_%s_%s.xml' % (freq, province.lower(), time.strftime(fmt), language)
-    return 'http://dd.weatheroffice.ec.gc.ca/observations/xml/%s/%s/%s' % (province.upper(), freq, fname)
+    fname = '%s_%s_%s_%s.xml' % (
```

Comment:
Can we change this to use Python's new style string formatting, while we're working on this?

Reply:
Yes, do you want it done just in this particular instance, or everywhere in the project?

Reply:
Just this instance I guess. Or... only in places where we're changing the code anyway. There shouldn't be too many of these left... we've been using str.format for a long time now.

Reply:
Ok, I'll have a look and see if I spot it anywhere else.